### DISCUSSION TITLE
DarkMode is not working on production

### DISCUSSION THREAD

**Discussion by ajaythakkar** [tailwindlabs/tailwindcss#13067] [Help]
**What version of Tailwind CSS are you using?**

v3.0.7

**What build tool (or framework if it abstracts the build tool) are you using?**

NuxtJs 3.7

I am using:
darkMode: 'class',

I have tried all the ways suggested in the past but it is not working.

safelist: ['dark'],

**Comment by adamwathan** [comment]
Hey! You’ll have to provide a reproduction for anyone to be able to help you. Can you create a small GitHub repo that others can clone to experience the problem?

**Reply by ajaythakkar** [reply]
Okay, I will do that, can you please tell me what is the best practice to prevent purging CSS with darkMode?

I local it is working fine, so it's just getting purge on production.

**Comment by ajaythakkar** [comment]
This works for me :

`darkMode: ['variant', '&:not(.light *)'],`
